phpunit.xml.dist

```diff
@@ -10,7 +10,6 @@
         convertNoticesToExceptions="true"
         convertWarningsToExceptions="true"
         convertDeprecationsToExceptions="true"
-        bootstrap="tests/phpunit/bootstrap.php"
```

Comment:
I wasn't able to recreate an instance where clearing cache was actually necessary to pick up changes in command files.